### PR TITLE
Fix technical typos and spelling

### PR DESCRIPTION
**Pull request by clinssen**
Small documentation fixes, collected while copying updated model documentation to NESTML following the merge of #1530.

### CODE REVIEW

File: models/hh_cond_beta_gap_traub.h [hunk 1]
```diff
@@ -89,19 +89,18 @@ to match those used with NEST 1.9.10 when preparing data for [1]_. Code for all
...
-Traub and Miles used \f$ t_ref = 3 ms \f$ [2, p 118], while we used
-\f$ t_ref = 2 ms \f$ in [2]_.
+Traub and Miles used :math:`t_{ref} = 3` ms ([2]_, p 118), while we used
+:math:`t_{ref} = 2` ms in [2]_.
 
 Post-synaptic currents
 Incoming spike events induce a post-synaptic change of conductance modelled by a
-beta function as outlined in [4,5]. The beta function is normalised such that an
-event of weight 1.0 results in a peak current of 1 nS at \f$ t = tau_rise_xx \f$
+beta function as outlined in [4]_ [5]_. The beta function is normalised such that an
```
Comment (sarakonradi) [inline models/hh_cond_beta_gap_traub.h:102]: Could we change the few instances of "normalised" to "normalized"?

Comment (clinssen) [reply models/hh_cond_beta_gap_traub.h]: Done!

**Comment by clinssen** [comment]
Totally on board rewriting "Remarks" to a header or a ReadTheDocs admonition box. But perhaps that should be a separate PR?

File: models/hh_cond_beta_gap_traub.h [hunk 2]
```diff
@@ -89,19 +89,18 @@ to match those used with NEST 1.9.10 when preparing data for [1]_. Code for all
 simulators covered is available from ModelDB [3]_.
 
 Note:
-In this model, a spike is emitted if
-
- :math:`V_m >= V_T + 30 mV` and `V_m` has fallen during the current time step
+In this model, a spike is emitted if :math:`V_m \geq V_T + 30` mV and :math:`V_m` has fallen during the current time
+step.
```
Comment (jougs) [inline models/hh_cond_beta_gap_traub.h:93]: I'd prefer if lines were broken at a reasonable length to ease reading also in the code files.

File: models/iaf_cond_exp_sfa_rr.h [hunk 3]
```diff
@@ -61,25 +61,19 @@ extern "C" int iaf_cond_exp_sfa_rr_dynamics( double, const double*, double*, voi
 Short description
 +++++++++++++++++
 
-Simple conductance based leaky integrate-and-fire neuron model
+Conductance based leaky integrate-and-fire model with spike-frequency adaptation and relative refractory mechanisms
```
Comment (jougs) [inline models/iaf_cond_exp_sfa_rr.h:64]: Break line.

File: models/iaf_cond_exp_sfa_rr.h [hunk 4]
```diff
@@ -61,25 +61,19 @@ extern "C" int iaf_cond_exp_sfa_rr_dynamics( double, const double*, double*, voi
...
 
 Description
 +++++++++++
 
-iaf_cond_exp_sfa_rr is an iaf_cond_exp_sfa_rr i.e. an implementation of a
-spiking neuron using IAF dynamics with conductance-based synapses,
-with additional spike-frequency adaptation and relative refractory
-mechanisms as described in Dayan+Abbott, 2001, page 166.
+iaf_cond_exp_sfa_rr is an implementation of a spiking neuron using integrate-and-fire dynamics with conductance-based
+synapses, with additional spike-frequency adaptation and relative refractory mechanisms as described in [2]_, page 166.
```
Comment (jougs) [inline models/iaf_cond_exp_sfa_rr.h:70]: Break lines.

File: models/iaf_cond_exp_sfa_rr.h [hunk 5]
```diff
@@ -61,25 +61,19 @@ extern "C" int iaf_cond_exp_sfa_rr_dynamics( double, const double*, double*, voi
...
 
-As for the iaf_cond_exp_sfa_rr, Incoming spike events induce a post-synaptic
-change  of  conductance  modelled  by an  exponential  function.  The
-exponential function is  normalised such that an event  of weight 1.0
-results in a peak current of 1 nS.
+Incoming spike events induce a post-synaptic change of conductance modelled by an exponential function. The exponential
+function is normalized such that an event of weight 1.0 results in a peak current of 1 nS.
```
Comment (jougs) [inline models/iaf_cond_exp_sfa_rr.h:73]: Break lines.

File: models/iaf_cond_exp_sfa_rr.h [hunk 6]
```diff
@@ -61,25 +61,19 @@ extern "C" int iaf_cond_exp_sfa_rr_dynamics( double, const double*, double*, voi
...
 
-Outgoing spike events induce a change of the adaptation and relative
-refractory conductances by q_sfa and q_rr, respectively.  Otherwise
-these conductances decay exponentially with time constants tau_sfa
-and tau_rr, respectively.
+Outgoing spike events induce a change of the adaptation and relative refractory conductances by q_sfa and q_rr,
+respectively. Otherwise these conductances decay exponentially with time constants tau_sfa and tau_rr, respectively.
```
Comment (jougs) [inline models/iaf_cond_exp_sfa_rr.h:76]: Break lines.

File: models/izhikevich.h [hunk 7]
```diff
@@ -49,28 +49,29 @@ Implementation of the simple spiking neuron model introduced by Izhikevich
...
+As published in [1]_, the numerics differs from the standard forward Euler technique in two ways:
 
-As published in [1]_, the numerics differs from the standard forward Euler
-technique in two ways:
-1) the new value of u is calculated based on the new value of v, rather than
-the previous value
-2) the variable v is updated using a time step half the size of that used to
-update variable u.
+1) the new value of :math:`u` is calculated based on the new value of :math:`V_m`, rather than the previous value
+2) the variable :math:`V_m` is updated using a time step half the size of that used to update variable :math:`u`.
```
Comment (jougs) [inline models/izhikevich.h:67]: Break lines.

File: models/hh_cond_beta_gap_traub.h (hunk 2 above)
Comment (clinssen) [reply models/hh_cond_beta_gap_traub.h:93]: All the new or changed lines are wrapped at 120 characters, should we keep wrapping at 80 characters in the RST blocks to keep them internally consistent?

Comment (jougs) [reply models/hh_cond_beta_gap_traub.h:93]: Exactly!

Comment (jougs) [reply models/hh_cond_beta_gap_traub.h:93]: Also, because `Alt`+`Q` does it like this in Emacs ;-)

Comment (clinssen) [reply models/hh_cond_beta_gap_traub.h]: The line length debate is not dead yet! (Will it ever?) Let's save this for a rainy day.

Reformatting: done!

**Pull request merged**